Friendly-plans/app/src/androidTest/java/pg/autyzm/friendly_plans/TaskCreateActivityTest.java

```diff
@@ -101,8 +134,56 @@ public void When_AddingNewTask_Expect_NewTaskAddedToDB() throws InterruptedExcep
         assertThat(taskTemplates.get(0).getDurationTime(), is(1));
     }
 
+    @Test
+    public void When_AddingNewTask_and_NameIsEmpty_Expect_Warning() throws InterruptedException {
+        closeKeyboard(); // why should we use closeKeyboard() instead of predefined closeSoftKeyboard()?!
+        onView(withId(R.id.id_btn_task_next))
+                .perform(click());
+
+        onView(withId(R.id.id_et_task_name)).check
+                (matches(hasErrorText(
+                        activityRule.getActivity().getString(R.string.not_empty_msg))));
+    }
+
+    @Test
+    public void When_AddingNewTask_and_DurationIsEmpty_Expect_Warning() {
+        onView(withId(R.id.id_et_task_name)).
+                perform(typeText(GOOD_TASK_NAME));
+
+        onView(withId(R.id.id_btn_task_next)).perform(scrollTo());
+
+        onView(withId(R.id.id_btn_task_next))
+                .perform(click());
+
+        onView(withId(R.id.id_et_task_duration_time)).check
+                (matches(hasErrorText(
+                        activityRule.getActivity().getString(R.string.not_empty_msg))));
+    }
+
+    @Test
+    public void When_AddingNewTask_and_Name_Has_ForbiddenSymbols_Expect_Warning()
+            throws InterruptedException {
+        onView(withId(R.id.id_et_task_name)).
+                perform(typeText(BAD_TASK_NAME));
+        closeKeyboard();
+
+        onView(withId(R.id.id_btn_task_next))
+                .perform(click());
+
+        onView(withId(R.id.id_et_task_name)).check
+                (matches(hasErrorText(
+                        activityRule.getActivity().getString(R.string.only_letters_msg))));
+    }
+
+    @Test
+    public void When_AddingNewTask_and_Name_Already_Exists_Expect_Warning() {
+        // I have to emulate existing name on DB. How ?! I need smth  like
```

Comment:
Yes, right, This test should be added with task, #119. So you do no need this now.